scripts/zones/Lower_Jeuno/Zone.lua

```diff
@@ -81,16 +84,42 @@ end;
 
 function onGameHour()
     local VanadielHour = VanadielHour();
-
+    local playerOnQuestId = GetServerVariable("[JEUNO]CommService");
```

Comment:
can probably store this as localVar in one of the quest npc unless it needs to persist after server crash (death to all server vars)